### PR TITLE
ZEPPELIN-5690: use jinjava 2.5.4 due to cves

### PR DESCRIPTION
**Pull request by pjfanning**
### What is this PR for?

use jinjava 2.5.4 due to cves

### What type of PR is it?
Bug Fix

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN/ZEPPELIN-5690

### How should this be tested?
* Strongly recommended: add automated unit tests for any new or changed behavior
* Outline any manual steps to test the PR here.

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

**Comment by pjfanning** [comment]
Build affected by https://github.com/twitter/hadoop-lzo/issues/148

File: scalding/pom.xml [hunk 1]
```diff
@@ -47,6 +47,8 @@
       <name>Concurrent Maven Repo</name>
       <url>https://conjars.org/repo</url>
     </repository>
+
+    <!-- the twitter repo is unreliable (https://github.com/twitter/hadoop-lzo/issues/148) -->
```
Comment (jongyoul) [inline scalding/pom.xml:51]: In my understanding, Scalding is no longer under development. So I think we'd better remove this module itself. WDYT?

Comment (pjfanning) [reply scalding/pom.xml:51]: @jongyoul jinjava is also used in submarine module - would it be possible to treat the possible removal of the scalding module as a separate issue?

Comment (jongyoul) [reply scalding/pom.xml:51]: Of course, I just would like to say you don't have to consider the scalding module carefully. :-) I'll make a new PR to remove Scalding module itself.

**Comment by zjffdu** [comment]
LGTM

**Review by jongyoul — APPROVED** [review]

**Pull request merged**